sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/collectionOperations.scala

```diff
@@ -308,6 +308,170 @@ case class MapEntries(child: Expression) extends UnaryExpression with ExpectsInp
   override def prettyName: String = "map_entries"
 }
 
+/**
+ * Returns the union of all the given maps.
+ */
+@ExpressionDescription(
+usage = "_FUNC_(map, ...) - Returns the union of all the given maps",
```

Comment:
nit: indent

Reply:
@ueshin 

>We don't need to care about key duplication like CreateMap for now.

Just verifying: This means I should simply concatenate the maps, possibly creating additional duplicates.

Reply:
Yes, currently we don't check the key duplication in other functions as well. We should follow the behavior for now. We might need to discuss the issue in the future, so let's revisit the issue here at that time.